stubs/inertia/resources/js/Jetstream/Modal.vue

```diff
@@ -71,10 +73,6 @@
             }
 
             document.addEventListener('keydown', closeOnEscape)
-
-            this.$once('hook:destroyed', () => {
```

Comment:
However we solve this in the `Dropdown.vue` component, needs to also be done here.